meta/template.md

```diff
@@ -1,48 +1,37 @@
 
-# Spec Name
+# Title: A human-readable title for this proposal
 
-* Status: In Review / Reviewed / Implementing / Implemented
-* Author(s): [Author Name](https://github.com/NuGet)
-* Issue: [1](https://github.com/NuGet/Home/issues/1) Issue Title
+[Author Name](https://github.com/<username>) | [GitHub Issue](https://github.com/NuGet/Home/issues/<issue>)
 
-## Problem Background
+## Description: What is it?
```

Comment:
They are definitely really closely related. 

Description -> Tell me a paragraph about it
Problem -> Tell me what problems it solves
Why -> Tell me why we should solve this problem

These three really help define the motivation & prioritization from the maintainer perspective in my opinion.

I think they can definitely be grouped, I just worry that we would imply people would know how to put that all under one section, so I broke it down like I usually do.

Reply:
We can maybe consider Header 3 sections that way it's communicated that they are related. 

> I think they can definitely be grouped, I just worry that we would imply people would know how to put that all under one section, so I broke it down like I usually do.

I don't think there's harm in that :) I think it makes sense to break them in this particular way to an extent. I just see this and see a lot of `1 line` why sections, because the problem kind of covered that. 

My proposal would be, include the breakdown, but not by means of section headings, rather comments (either `<!-- here be comment -->` or just regular inline comment.

Reply:
I'll include comments for the explanation sections in a new revision.